### ISSUE TITLE
Schema fields Group By

### ISSUE THREAD

**Issue by MNoya** [GetDotaStats/site#226]
I was thinking for what schema values I would require for DotaCraft, and the first thing that came to mind, inspired by [WC3 Ladder page](http://classic.battle.net/war3/ladder/reports/last-week/Northrend/w3xp-reports-hero-usage.shtml) is Hero Usage, but obviously this data doesn't make sense on its own because it depends on the race picked.

I think there is two ways of approaching this, the ugly way is having one value for each possible combination (in this case it would be 12 values: first-second-third hero per race) and just send an empty string for the fields that aren't applicable (in this case, at least 9 fields will be empty).

The other way could be adding a "Group By" on the schema creation, and making one pie chart for each of the grouped values

![img](http://puu.sh/lttqg/9c4fac4bd4.png)

**Comment by jimmydorry** [comment]
Yep, that has been the next thing to implement in the `Mod overview page` for a while. It was never split out into its own issue though.
